modules/juce_core/native/juce_win32_SystemStats.cpp

```diff
@@ -144,6 +144,7 @@ void CPUInformation::initialise() noexcept
     hasSSE2  = (info[3] & (1 << 26)) != 0;
     hasSSE3  = (info[2] & (1 <<  0)) != 0;
     hasAVX   = (info[2] & (1 << 28)) != 0;
+    hasFMA3  = (info[2] & (1 << 12)) != 0;

```

Comment:
You haven't checked for FMA4 here at all?

Reply:
Indeed. I haven't found a way to do it, even by following the official Microsoft example for [`__cpuid`](https://docs.microsoft.com/en-us/cpp/intrinsics/cpuid-cpuidex?view=vs-2017).  I'm all ears if you have any ideas!

Reply:
I would have a look in info[2] bit 16 of `callCPUID (info, 0x80000001)`

Reply:
Changed, and pushed.